r/R/compute.R

```diff
@@ -344,7 +344,7 @@ cast_options <- function(safe = TRUE, ...) {
 #' @return `NULL`, invisibly
 #' @export
 #'
-#' @examplesIf arrow_with_dataset()
+#' @examplesIf arrow_with_dataset() && identical(Sys.getenv("R_ARROW_COLLECT_WITH_UDF"), "true")
```

Comment:
Can revert this

Reply:
Oh I see, you want to make sure it doesn't run on CRAN. So how about this, so that we get the examples running on the docs without having to set other env vars?

```suggestion
#' @examplesIf arrow_with_dataset() && identical(Sys.getenv("NOT_CRAN"), "true")
```